CHANGELOG.md

```diff
@@ -2,10 +2,14 @@
 
 ## Next Release
 ### Internal
+- [In Progress: 11 Nov 2020] Migrate app screens to use ViewBinding
```

Comment:
We are already tracking this with ` - [In Progress: 30 Nov 2020] Migrate app to use ViewBinding `. So, please remove this.